frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -300,8 +298,6 @@ const QueueItem: any = ({
 
   const [removePatientId, setRemovePatientId] = useState<string>();
 
-  let forceSubmit = false;
```

Comment:
This was odd to find and could explain some unpredictable/hard to replicate behavior